owl/SOMA-OBJ.owl

```diff
@@ -975,6 +975,9 @@
 
     <owl:Class rdf:about="http://www.ease-crc.org/ont/SOMA-OBJ.owl#ArtificialAgent">
         <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#PhysicalAgent"/>
+        <rdfs:comment>A physical objects with artificial characteristics, which can perform actions to achieve desired goals, typically has sensors and/or actuators.
```

Comment:
you mix up two pull requests here.
better use two *separate* branches in case you do two separate things